### PR TITLE
Add setting to trim logs on success

### PR DESCRIPTION
**Pull request by rystsov**
- Add setting to remove all TRACE and DEBUG logging on success to reduce space consumption
 - Use trace level by default

### CODE REVIEW

File: control/trim_logs.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    tmp_file = join(sys.argv[1], "tmp")
+
+    keep = True
+
+    with open(log_name, "r") as log_in:
+        with open(tmp_file, "w") as log_out:
+            for line in log_in:
+                if any(map(lambda x: line.startswith(x), ["TRACE", "DEBUG"])):
+                    keep = False
+                if any(map(lambda x: line.startswith(x), ["INFO", "WARN", "ERROR"])):
```
Comment (bharathv) [inline control/trim_logs.py:18]: nit: Do we have multi line messages? (ones that don't start with anything?)

Probably default keep = True and do keep &= not any(map(lambda x: line.startswith(x), ["TRACE", "DEBUG"]))

That way we retain everything except that ones beginning with TRACE/DEBUG.

File: control/trim_logs.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+                    keep = True
+                if keep:
+                    log_out.write(line)
+
+    remove(log_name)
```
Comment (bharathv) [inline control/trim_logs.py:23]: may needs a try except (remove tmp file in all cases)

**Review by bharathv — APPROVED** [review]
a nitpick but otherwise lgtm.

File: control/trim_logs.py (hunk 1 above)
Comment (rystsov) [reply control/trim_logs.py:18]: yes, when we log exceptions they occupy multiple lines

**Pull request merged**